### ISSUE TITLE
As of 2.4.0 can't use some .rx2 keys anymore (example "correlation" on factanal object)

### ISSUE THREAD

**Issue by lgautier** [rpy2/rpy2#229]
**[Original report](https://bitbucket.org/rpy2/rpy2/issue/219) by Horea Christian (Bitbucket: [TheChymera](https://bitbucket.org/TheChymera), GitHub: [TheChymera](https://github.com/TheChymera)).**

----------------------------------------

Please refer to [this StackOverflow question](http://stackoverflow.com/questions/25036588/extract-correlation-matrix-from-rs-factanal-via-rpy) for the full details.

In short, for:

```
#!python

from rpy2.robjects import r, numpy2ri
numpy2ri.activate()
fit = r.factanal(A, 5, rotation='promax')
```

I can't use:

```
#!python

corr = r.correlation(fit)
```

or 

```
#!python

corr = fit.rx2('correlation')
```

And I am forced to use:

```
#!python

corr = fit[3]
```

This is especially strange since:

```
#!python

load = r.loadings(fit)

```

works
